### PR TITLE
Add CC deployment example with goals configured

### PR DESCRIPTION
**Pull request by kyguy**
### Type of change

- Enhancement / new feature
- Documentation

### Description

- Adds new Cruise Control deployment example to demonstrate how to add custom goal lists
- Adds minor update to optimization goals documentation

### Checklist

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [x] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Review by scholzj — CHANGES_REQUESTED** [review]
You have to make the changes to `packaging/examples`

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Review by mimaison — APPROVED** [review]

**Review by PaulRMellor — APPROVED** [review]
Looks good. I made a suggestion on the note.

File: documentation/modules/cruise-control/con-optimization-goals.adoc [hunk 1]
```diff
@@ -134,7 +134,7 @@ You configure main optimization goals, if necessary, in the Cruise Control deplo
 
 * If you need to modify the inherited main optimization goals, specify a list of goals, in descending priority order, in the `goals` configuration option.
 
-NOTE: If you change the inherited main optimization goals, you must ensure that the hard goals, if configured in the `hard.goals` property in `Kafka.spec.cruiseControl.config`, are a subset of the main optimization goals that you configured. Otherwise, errors will occur when generating optimization proposals.
+NOTE: If you change the inherited main optimization goals or default goals, you must ensure that the hard goals, if configured in the `hard.goals` property in `Kafka.spec.cruiseControl.config`, are a subset of the main optimization goals and default goals that you configured. Otherwise, errors will occur when generating optimization proposals.
```
[the remark this answers is not in the document]
Comment (kyguy) [reply documentation/modules/cruise-control/con-optimization-goals.adoc]: Thanks Paul, added your suggestion

**Comment by kyguy** [comment]
This PR will be ready to merge once the tests pass successfully

**Pull request merged**